scripts/ci/kubernetes/docker/build.sh

```diff
@@ -34,8 +34,18 @@ fi
 echo "Airflow directory $AIRFLOW_ROOT"
 echo "Airflow Docker directory $DIRNAME"
 
+if [[ ${PYTHON_VERSION} == '3' ]]; then
+  PYTHON_DOCKER_IMAGE=python:3.6-slim
+else
+  PYTHON_DOCKER_IMAGE=python:2.7-slim
+fi
+
 cd $AIRFLOW_ROOT
-python setup.py sdist -q
+docker run -ti --rm -e SLUGIFY_USES_TEXT_UNIDECODE -v ${AIRFLOW_ROOT}:/airflow \
+    -w /airflow ${PYTHON_DOCKER_IMAGE} ./scripts/ci/kubernetes/docker/compile.sh
```

Comment:
compile airflow in a container, in addition it will compile the web assets

Reply:
This makes a lot of sense. Thanks for setting this up.